packages/contracts/scripts/write-constants.ts

```diff
@@ -57,6 +60,7 @@ const writeConstants = async () => {
     `pragma solidity >=0.6.2 <0.9.0;\n\n` +
     `import { NetworkInfo, NetworkType } from "./SphinxPluginTypes.sol";\n\n` +
     `contract SphinxConstants {\n` +
+    `  string public constant sphinxVersion = 'v${version}';\n` +
```

Comment:
Including this here makes it easy for us to fetch the current version of the Solidity contracts library.